x-pack/test/detection_engine_api_integration/security_and_spaces/rule_execution_logic/query.ts

```diff
@@ -673,6 +679,428 @@ export default ({ getService }: FtrProviderContext) => {
           [ALERT_SUPPRESSION_DOCS_COUNT]: 16,
         });
       });
+
+      it('should correctly deduplicate null values across rule runs', async () => {
```

Comment:
Appreciate all the different FTR tests here -- thank you!

Would there be any benefit to some additional scenarios around `max_signals` boundaries or when a `timestamp_override` kicks in? Perhaps the existing timestamp test you have covers the latter already?

Reply:
Good point, I added a test each for `max_signals` and `timestamp_override` specific to alert suppression